### PR TITLE
Update default tag in Helm charts for 1.2

### PR DESCRIPTION
**Pull request by rlenglet**

### CODE REVIEW

File: install/kubernetes/helm/istio-init/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ global:
   hub: gcr.io/istio-release
 
   # Default tag for Istio images.
-  tag: master-latest-daily
+  tag: release-1.2-latest-daily
```
Comment (sdake) [inline install/kubernetes/helm/istio-init/values.yaml]: have to take care with this tag - there is related code in the infrastructure that does a sed replacement on it.  Has an accompanying PR been submitted to test-infra?

**Comment by sdake** [comment]
feel free to remove my hold if the test infra has already been updated and I am unaware of it.

Cheers
-steve

File: install/kubernetes/helm/istio-init/values.yaml (hunk 1 above)
Comment (fpesce) [reply install/kubernetes/helm/istio-init/values.yaml]: I'd like @rlenglet to chime in on this one.

Comment (rlenglet) [reply install/kubernetes/helm/istio-init/values.yaml]: @sdake Branch `release-1.1` uses tag `release-1.1-latest-daily`. Why would `release-1.2` be different?

Comment (rlenglet) [reply install/kubernetes/helm/istio-init/values.yaml]: @sdake if you were referring to `release/gcb/helm_values.sh`, it should already work:
```
sed -i "s|tag: .*-latest-daily|tag: ${CB_VERSION}|g"         ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio*/values.yaml
```

**Comment by rlenglet** [comment]
@sdake no further change is needed, cf. `release/gcb/helm_values.sh`:
```
sed -i "s|tag: .*-latest-daily|tag: ${CB_VERSION}|g"         ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio*/values.yaml
```

**Comment by sdake** [comment]
Looks like infra is generally compatible with this change.

https://github.com/istio/cni/pull/125#issuecomment-502182663

**Pull request merged**